storybookjs/storybook
# Build: Give linting sub-commands a directory to run against

Issue: https://github.com/storybookjs/storybook/issues/17542

## What I did
I was very confused when running `yarn lint:js` did not result in any files being linted.  I've now realized that the target (`.`) is specified in the parent command, `yarn lint`.  It seems to me like commands should do the thing they say they're doing, without needing extra arguments in order to function.  So, I've moved the `.` into the linting subcommands here.  I'm happy to close this out if there are reasons to keep it apart, but it was definitely confusing to me.

## How to test


You can run `yarn lint:js --debug` to verify that it is indeed running against files now.
